storage/rocksdb/clone/donor.cc

```diff
@@ -102,10 +105,14 @@ class [[nodiscard]] rdb_checkpoint final {
  private:
   const std::string m_dir;
 
+  std::string m_full_dir;
+
   bool m_active = false;
 
   static std::atomic<std::uint64_t> m_next_id;
 
+  std::atomic<std::uint64_t> m_next_sub_id{1};
```

Comment:
This should not be atomic. It is private to a single clone session, and all its accesses happen with `m_donor_mutex` locked.

Reply:
resolved